### PR TITLE
Enable painting segmentations and increase slicing performance 

### PR DESCRIPTION
**Pull request by uermel**
1. make launchable from within napari
2. load segmentations with 'r+'
3. use napari-ome-zarr for tomogram to increase performance

### CODE REVIEW

File: src/napari_copick/widget.py [hunk 1]
```diff
@@ -171,36 +175,20 @@ def lazy_load_voxel_spacing(self, item, voxel_spacing):
...
-        zarr_path = tomogram.zarr()
-        zarr_group = zarr.open(zarr_path, "r")
-
-        # Determine the number of scale levels
-        scale_levels = [key for key in zarr_group.keys() if key.isdigit()]
-        scale_levels.sort(key=int)
+        zarr_path = tomogram.zarr().path
 
-        data = [zarr_group[level] for level in scale_levels]
+        self.viewer.open(zarr_path, plugin="napari-ome-zarr")
```
Comment (uermel) [inline src/napari_copick/widget.py:180]: makes use of multiscale and is more performant.

File: src/napari_copick/widget.py [hunk 2]
```diff
@@ -171,36 +175,20 @@ def lazy_load_voxel_spacing(self, item, voxel_spacing):
...
-        self.viewer.add_image(
-            data, name=f"Tomogram: {tomogram.meta.tomo_type}", scale=scale
-        )
         self.info_label.setText(
             f"Loaded Tomogram: {tomogram.meta.tomo_type} with num scales = {len(scale_levels)}"
         )
 
     def load_segmentation(self, segmentation):
-        zarr_data = zarr.open(segmentation.zarr().path, "r")
+        zarr_data = zarr.open(segmentation.zarr().path, "r+")
```
Comment (uermel) [inline src/napari_copick/widget.py:187]: Make writable.

File: src/napari_copick/widget.py [hunk 3]
```diff
@@ -24,9 +24,13 @@
...
 class CopickPlugin(QWidget):
-    def __init__(self, viewer, config_path=None):
+    def __init__(self, viewer=None, config_path=None):
         super().__init__()
-        self.viewer = viewer
+        if viewer:
+            self.viewer = viewer
+        else:
+            self.viewer = napari.Viewer()
+
```
Comment (uermel) [inline src/napari_copick/widget.py:33]: Retains the ability to call from shell, but also allows starting from within Napari.

**Comment by uermel** [comment]
Ooops, the failing tests seem to be examples from the cookie cutter.

**Review by kephale — APPROVED** [review]
Great, looks good!

**Pull request merged**